discourse/janitor.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Discourse",
+  "description": "Discourse is the 100% open source discussion platform built for the next decade of the Internet.",
+  "icon": "https://janitor.technology/img/discourse.png",
+  "docker": {
+    "image": "janx/discourse"
```

Comment:
Nit: Please use our new Docker Hub org, i.e. `janitortechnology/discourse`, instead of my personal repository. 🙂